cpp/box/Box.h

```diff
@@ -5,10 +5,10 @@
 #define BOX_H
 
 #include "utils.h"
+#include <algorithm>
 #include <complex>
 #include <sstream>
 #include <stdexcept>
-#include <algorithm>
 
```

Comment:
`clang-format` has a preferred sort order for includes (alphabetical). Up to now reformatting has only been something we do on releases, and this include was probably added since the last release by #590.